utils/fixed_decimal/src/decimal.rs

```diff
@@ -3167,10 +2527,6 @@ fn test_from_str_scientific() {
         pub output: &'static str,
     }
     let cases = [
-        TestCase {
-            input_str: "-5.4e10",
```

Comment:
Suggestion: Add back some of these cases as failure cases to demonstrate that UnsignedFixedDecimal will fail to parse strings with a minus sign